config.lua.dist

```diff
@@ -42,6 +42,7 @@ deathLosePercent = -1
 -- NOTE: set housePriceEachSQM to -1 to disable the ingame buy house functionality
 housePriceEachSQM = 1000
 houseRentPeriod = "never"
+houseAccountOwner = false
```

Comment:
Maybe `houseOwnedByAccount` would convey the meaning better?

Reply:
> Maybe `houseOwnedByAccount` would convey the meaning better?
Yes, totally better!

Reply:
I think you forgot to update this file.